examples/step-63/doc/intro.dox

```diff
@@ -1,9 +1,181 @@
 <br>
 
-<i>This program was contributed by Thomas Clevenger and Timo Heister.</i>
+<i>This program was contributed by Thomas C. Clevenger and Timo Heister.
+
+The creation of this tutorial was partially supported by NSF Award
+DMS-1522191, DMS-1901529, OAC-1835452, by the Computational
+Infrastructure in Geodynamics initiative (CIG), through the NSF under
+Award EAR-0949446 and EAR-1550901 and The University of California -
+Davis, and by Technical Data Analysis, Inc. through US Navy STTR
+N16A-T003.
```

Comment:
Is this actually true? I'll note that if you state that they supported the work, then they might have copyright restrictions on the use of the work you have produced as part of the contract.

Reply:
I do have the permission to work on related deal.II features, but maybe this is not so important here...